index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Project</title>
+    <link rel="stylesheet" href="styles/app.css">
+  </head>
+  <body>
+    <div class="header">
+      <div class="header__left">
+        <h1>Popular Github</h1>
+      </div>
+      <div class="header__right">
+        <p>people</p>
+        <p>repos</p>
+      </div>
+    </div>
+
+    <main>
+      <div class="profiles">
+        <div class="profiles__item-1">
```

Comment:
Don't individually number classes. We'll review this today but a big part of using classes is that they are reusable and repeatable. All of the profiles should be able to be the same class and just work :tm:.